RepoDb.Core/RepoDb/QueryField/ParseExpression.cs

```diff
@@ -443,7 +431,7 @@ internal static ClassProperty GetProperty<TEntity>(Expression expression)
         /// <returns></returns>
         internal static ClassProperty GetProperty<TEntity>(LambdaExpression expression)
             where TEntity : class =>
-            GetProperty<TEntity>(expression.ToLambda().Body);
+            GetProperty<TEntity>(expression.Body);
```

Comment:
Is this working?

Reply:
Yes, It was same cast. LambdaExpression was cast to LambdaExpression.